blivet/devicelibs/dasd.py

```diff
@@ -298,15 +298,18 @@ def write_dasd_conf(disks, root):
     # this does catch every DASD, even non-aliases, but we're only going to be
     # checking for a very specific flag, so there won't be any duplicate entries
     # in dasd.conf
-    devs = [d for d in os.listdir(sysfs) if d.startswith("0.0")]
-    with open(os.path.realpath(root + "/etc/dasd.conf"), "a") as f:
-        for d in devs:
-            aliasfile = "%s/%s/alias" % (sysfs, d)
-            with open(aliasfile, "r") as falias:
-                alias = falias.read().strip()
-
-            # if alias == 1, then the device is an alias; otherwise it is a
-            # normal dasd (alias == 0) and we can skip it, since it will have
-            # been added to dasd.conf in the above block of code
-            if alias == "1":
-                f.write("%s\n" % d)
+    try:
+        devs = [d for d in os.listdir(sysfs) if d.startswith("0.0")]
+        with open(os.path.realpath(root + "/etc/dasd.conf"), "a") as f:
+            for d in devs:
+                aliasfile = "%s/%s/alias" % (sysfs, d)
+                with open(aliasfile, "r") as falias:
+                    alias = falias.read().strip()
+
+                # if alias == 1, then the device is an alias; otherwise it is a
+                # normal dasd (alias == 0) and we can skip it, since it will have
+                # been added to dasd.conf in the above block of code
+                if alias == "1":
+                    f.write("%s\n" % d)
+    except OSError:
+        pass
```

Comment:
I don't think this is the right thing to do here as it may hide real issues. Is the real problem here that the config file doesn't exist or what? If so, test the file existence before trying to append to it and add a comment why it is okay if it doesn't exist. Or is it the problem with some of the ``/alias`` files? If that's the case, there should again be a check before trying to open it with an explanation in a comment.